### PR TITLE
[release/3.0] Getting ready for Teatro

### PR DESCRIPTION
**Pull request by marutosi**
Description in commit logs.

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage. <a href='http://marutosi-finn-release-3-0-teatro.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Click to open.</a>

**Comment by linki** [comment]
enabling teatro for `release/3.0` needs at least some discussion. @machisuji @ulferts @myabc @tessi do we want that?

File: .teatro.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+stage:
+  before:
+    - bower install --allow-root
```
Comment (tessi) [inline .teatro.yml:3]: This won't work in 3.0.x

**Comment by tessi** [comment]
In principle, I think having teatro in 3.0x won't do us no harm. We could always start a teatro vm to feel nostalgic.

File: .teatro.yml (hunk 1 above)
Comment (linki) [inline .teatro.yml:3]: @marutosi ping

**Comment by divineforest** [comment]
Do you need help with Teatro?

**Comment by tessi** [comment]
@marutosi if you remove the `bower install` line (on which I posted a comment earlier) and if you use the current `.teatro.yml` file from the `dev` branch as a base (which has a more specific foreman command), this PR can be merged.

@divineforest Thanks for offering a helping hand! While you're here, I have a question unrelated to this PR: Is it somehow possible to add the @TeatroIO comment without sending a notification email? It kind of spams the mail inbox. (I'm not aware of a way to avoid the mail, but maybe it helps to contact the github folks). Thanks for offering teatro. I realy like it.

**Comment by marutosi** [comment]
> if you remove the bower install line

Why will not you fix?

**Comment by linki** [comment]
@marutosi your pr, your responsibility

**Comment by marutosi** [comment]
> your responsibility

If you think so, give me pull request.

**Comment by linki** [comment]
pardon?

**Comment by marutosi** [comment]
It is DVCS way.

**Comment by linki** [comment]
@marutosi fix coming or not?

**Comment by marutosi** [comment]
If you want to fix, give me pull request.

Git is DVCS.
I don't want to record my name as commiter in revision which I don't know it is correct or not.

**Comment by linki** [comment]
no worries, we don't bite if it doesn't work the first time

**Comment by linki** [comment]
it's so quite out here. any progress on this PR soon, @marutosi? otherwise I would like close it.

**Comment by marutosi** [comment]
> any progress

I am still waiting your PR.

**Comment by myabc** [comment]
I don't think it's worthwhile pursuing this right now.

**Pull request closed without merging**